Makefile

```diff
@@ -43,4 +55,4 @@ docker-push:
 	docker push digitalmarketplace/api:${RELEASE_NAME}
 
 
-.PHONY: virtualenv requirements requirements_for_test test_pep8 test_migrations test_unit test test_all run_migrations run_app run_all docker-build docker-push
+.PHONY: virtualenv requirements requirements-dev freeze-requirements test_pep8 test_migrations test_unit test test_all run_migrations run_app run_all docker-build docker-push
```

Comment:
Consistency would be nice if I'm finally going to have to start using the `make` commands :)